app/assets/images/bg-wave.svg

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
what kind of language is this ?? To format the image ?

Reply:
SVG used for the background image. SVG is to graphics what HTML is to text

Reply:
Ok good to know